src/test/cost_matrix_test.cpp

```diff
@@ -0,0 +1,61 @@
+// #include "database/online_database.h"
+#include "database/cost_matrix.h"
+#include "localization_protos.pb.h"
+#include "test_utils.h"
+
+#include "gtest/gtest.h"
+#include <filesystem>
+
+class CostMatrixTest : public ::testing::Test {
+public:
+  void SetUp() {
```

Comment:
If you want to remove the file written here, you might want to add the `TearDown` function.

Reply:
Done.